Bevy 0.13 `RenderAssetUsages` fails to build due to `serde::Deserialize`.

## Bevy version

Bevy 0.13.0.

## \[Optional\] Relevant system information

If you cannot get Bevy to build or run on your machine, please include:

- Using Rust 1.76.0
- Windows 10

## What you did

Compiled my library which uses Bevy, with default features off, but with bevy_asset and bevy_render enabled.

## What went wrong

Failed to compile with the folling error:
```
error[E0277]: the trait bound `render_asset::_::InternalBitFlags: color::_::_serde::Serialize` is not satisfied
   --> E:\Programs\Cargo\registry\src\index.crates.io-6f17d22bba15001f\bevy_render-0.13.0\src\render_asset.rs:70:14
    |
53  | / bitflags::bitflags! {
54  | |     /// Defines where the asset will be used.
55  | |     ///
56  | |     /// If an asset is set to the `RENDER_WORLD` but not the `MAIN_WORLD`, the asset will be
...   |
70  | |     #[derive(Serialize, TypePath, Deserialize, Hash, Clone, Copy, PartialEq, Eq, Debug)]
    | |              ^^^^^^^^^ the trait `color::_::_serde::Serialize` is not implemented for `render_asset::_::InternalBitFlags`
...   |
74  | |     }
75  | | }
    | |_- required by a bound introduced by this call
    |
    = help: the following other types implement trait `color::_::_serde::Serialize`:
              bool
              char
              isize
              i8
              i16
              i32
              i64
              i128
            and 255 others
note: required by a bound in `serialize_newtype_struct`
   --> E:\Programs\Cargo\registry\src\index.crates.io-6f17d22bba15001f\serde-1.0.196\src\ser\mod.rs:900:12
    |
894 |     fn serialize_newtype_struct<T: ?Sized>(
    |        ------------------------ required by a bound in this associated function
...
900 |         T: Serialize;
    |            ^^^^^^^^^ required by this bound in `Serializer::serialize_newtype_struct`
    = note: this error originates in the derive macro `Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `render_asset::_::InternalBitFlags: Deserialize<'_>` is not satisfied
  --> E:\Programs\Cargo\registry\src\index.crates.io-6f17d22bba15001f\bevy_render-0.13.0\src\render_asset.rs:53:1
   |
53 | / bitflags::bitflags! {
54 | |     /// Defines where the asset will be used.
55 | |     ///
56 | |     /// If an asset is set to the `RENDER_WORLD` but not the `MAIN_WORLD`, the asset will be
...  |
74 | |     }
75 | | }
   | |_^ the trait `Deserialize<'_>` is not implemented for `render_asset::_::InternalBitFlags`
```

--- a/crates/bevy_render/Cargo.toml
+++ b/crates/bevy_render/Cargo.toml
@@ -74,7 +74,7 @@ wgpu = { version = "0.19.1", default-features = false, features = [
 ] }
 naga = { version = "0.19", features = ["wgsl-in"] }
 serde = { version = "1", features = ["derive"] }
-bitflags = "2.3"
+bitflags = { version = "2.3", features = ["serde"] }
 bytemuck = { version = "1.5", features = ["derive"] }
 downcast-rs = "1.2.0"
 thread_local = "1.1"

